src/frontend/Felles/Lenker/LenkeTilPersonopplysningsside.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode } from 'react';
+import { Link } from '@navikt/ds-react';
+import { ExternalLink } from '@navikt/ds-icons';
+import { Ressurs, RessursStatus } from '../../App/typer/ressurs';
+import { useApp } from '../../App/context/AppContext';
+import { BodyShortSmall } from '../Visningskomponenter/Tekster';
+import { EToast } from '../../App/typer/toast';
+
+interface IProps {
+    personIdent?: string;
+    children: ReactNode;
+}
+
+export const LenkeTilPersonopplysningsside: React.FC<IProps> = ({ personIdent, children }) => {
+    const { axiosRequest, settToast } = useApp();
+
+    const redirectTilPersonopplysningsside = (personIdent: string) => {
+        axiosRequest<string, { personIdent: string }>({
+            method: 'POST',
+            url: `/familie-ef-sak/api/fagsak-person/personx`,
```

Comment:
Litt usikker på om vi trenger `/person` etter fagsak-person for dette endepunktet 👀 
```suggestion
            url: `/familie-ef-sak/api/fagsak-person/person`,
```